libandroid-navigation/src/test/java/com/mapbox/services/android/navigation/v5/navigation/ElapsedTimeTest.java

```diff
@@ -8,19 +8,25 @@
 
 public class ElapsedTimeTest {
 
+  private static final double DELTA = 1E-10;
```

Comment:
We don't need such big `DELTA` especially if we're rounding to 2 point decimals and we're testing that the rounding is correct.